lib/features/home/presentation/home_controller.dart

```diff
@@ -90,7 +90,11 @@ class HomeController extends ReloadableController {
 
   static void downloadCallback(String id, DownloadTaskStatus status, int progress) {}
 
-  void _cleanupCache() async {
+  Future<void> _cleanupCache() async {
+    if (PlatformInfo.isIOS) {
+      _handleUserClickNotificationOnIOS();
+    }
```

Comment:
no better place to add this function?

Reply:
IMO, it is the most suitable place for us to get the content from the notification when the app starts launching